### ISSUE TITLE
Intel - Natty:  Partial screenshot capture with red border

### ISSUE THREAD

**Issue by craige111** [zachtib/lookit#27]
The problem with the red border that appears to work, but due to a bug, the partial screenshot you get includes (you guessed it) the red border itself round the edge. This is a fault.

**Comment by scribu** [comment]
Same problem here.

**Comment by zachtib** [comment]
Yep, confirmed. Looking into it for the next release when I get some time.

**Comment by stas** [comment]
It's tricky.
Cairo knows to stroke areas only with a border included, I couldn't find an option, to set border outside the rectangle.
